src/App.tsx

```diff
@@ -1,45 +1,87 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import './App.css';
-import Header from './components/Header';
+import { Innholdsformasjon } from './types';
 import CollapsedBox from './components/CollapsedBox';
-import { TitleLabel, AuthorLabel, ISBNLabel, ImageContainer } from './components/CustomElements';
-import pelsjegere from './resources/pelsjegere.jpg';
+import Header from './components/Header';
+import { ErrorTextField, ImageContainer, ISBNLabel, TitleLabel } from './components/CustomElements';
+import { getInnholdsinformasjon } from './services/api';
 
 const URL = window.location.href;
+const imageUrl = process.env.REACT_APP_INNHOLDSTJENESTE_IMAGES_URL;
+
+function isEmpty(array?: string[]): boolean {
+  return array === undefined || array.length < 1;
+}
 
 const App: FC = () => {
-  const shortSummary =
-    'Etter å ha brutt opp fra sin sakførerpraksis i Norge, reiste Ingstad til arktiske Canada, der han levde i fire år som pelsjeger.';
-  const longSummary =
-    'Etter å ha brutt opp fra sin sakførerpraksis i Norge, reiste Ingstad til arktiske Canada, der han levde i fire år som pelsjeger. Pelsjegerliv er beretningen om hans opplevelser i denne tiden. Boken ble utgitt i 1931  og er hans første av i alt tolv bøker';
-  const tableOfContents = [
-    'The opening',
-    'Second breakfast',
-    "Three's a crowd",
-    'May the fourth be with you',
-    'Cinco de mayo',
-  ];
+  const [innholdsinformasjon, setInnholdsinformasjon] = useState<Innholdsformasjon | undefined>();
+  const [errorPresent, setErrorPresent] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>('');
```

Comment:
Vi har gått over til å bruke Error-objektet i DLR. se https://github.com/BIBSYSDEV/dlr-gui-new/blob/4b53c6fc4c98534e9d7d03638ee9022326b7a400/src/pages/my_resources/MyResources.tsx#L46

Reply:
Tenker du at vi bør gjøre det også? Hva er oppsidene/nedsidene?

Reply:
da har man kun et objekt. ikke to seperate som egentlig hører sammen